CollAction/Frontend/app/global/MobileNavDraw.tsx

```diff
@@ -45,7 +48,7 @@ export default class MobileNavDraw extends React.Component<IMobileNavDrawProps,
     return(
       <div id="draw-menu">
         <div id="draw-menu-close-button" onClick={() => self.close()}>
-          <i className="fa fa-times" aria-hidden="true"></i>
+          <FontAwesomeIcon icon={faTimes} />
```

Comment:
Maybe it's a good idea to use normal html here? So: `<i class="fa fa-times"></i>` instead of using a React component here? That way we keep it a little bit in line with the cshtml files.

Reply:
It is due to performance improvement, v5 works different then v4, you can include single icon for page instead of downloading entire library.